upgrade/steps_helpers.go

```diff
@@ -442,18 +445,21 @@ func latestReleaseInfo(ctx context.Context, repo string) (releaseInfo, error) {
 	return info, err
 }
 
-var latestReleaseVersion = stepv2.Func11E("Latest Release Version",
-	func(ctx context.Context, repo string) (*semver.Version, error) {
+var latestReleaseVersion = stepv2.Func12E("Latest Release Version",
+	func(ctx context.Context, repo string) (*semver.Version, bool, error) {
```

Comment:
What does the bool signify?

Reply:
If a release was found. It's equivalent to checking if the `*semver.Version` is nil, but that felt like a footgun for go. Generally, for `(*T, error)`, I assume that `error == nil <=> *T != nil`